### PR TITLE
Dotnet SDK 2.7.0 release notes

### PR DESCRIPTION
**Pull request by jeffrymorris**

### CODE REVIEW

File: modules/ROOT/pages/relnotes-dotnet-sdk.adoc [hunk 1]
```diff
@@ -79,6 +79,45 @@ https://github.com/couchbase/couchbase-net-client/fork
...
+
+This release has the following known issues:
+
+* https://issues.couchbase.com/browse/NCBC-1296[NCBC-1296] -
+.NETCore SDK N1QL query throws System.PlatformNotSupportedException
+on some .NET Core platforms.  A workaround is available.
+* https://issues.couchbase.com/browse/NCBC-1502[NCBC-1502] -
+Fatal error: unexpected end of JSON input
+
+=== New Features and Behavioural Changes
```
Comment (ingenthr) [inline modules/ROOT/pages/relnotes-dotnet-sdk.adoc]: Should be en/US, right?  One too many 'u's.

File: modules/ROOT/pages/relnotes-dotnet-sdk.adoc [hunk 2]
```diff
@@ -79,6 +79,45 @@ https://github.com/couchbase/couchbase-net-client/fork
 After you have successfully built the source, it's then just a matter of referencing the binaries (.DLL files) from your consuming project. 
 _Note that you can checkout a specific tag for each release as well._
 
+== Version 2.7.0 (2nd October 2018)
+
+http://packages.couchbase.com/clients/net/2.5/Couchbase-Net-Client-2.7.0.zip[Download] | http://docs.couchbase.com/sdk-api/couchbase-net-client-2.7.0[API Reference]
+
+=== Known Issues
```
Comment (ingenthr) [inline modules/ROOT/pages/relnotes-dotnet-sdk.adoc:86]: Formatting seems off here.  We should start with new features, then cover fixes, then cover known issues, right?  I see it's not that way here already in the past though.  Can we make it consistent?  Java certainly leads with new features.  We can leave this for @RichardSmedley if need be, but it should be a quick fix.

File: modules/ROOT/pages/relnotes-dotnet-sdk.adoc [hunk 3]
```diff
@@ -79,6 +79,45 @@ https://github.com/couchbase/couchbase-net-client/fork
...
+* https://issues.couchbase.com/browse/NCBC-1502[NCBC-1502] -
+Fatal error: unexpected end of JSON input
+
+=== New Features and Behavioural Changes
+
+* https://issues.couchbase.com/browse/NCBC-1652[NCBC-1652] -         Update build scripts to new standard &amp; support non-windows
+* https://issues.couchbase.com/browse/NCBC-1758[NCBC-1758] -         Add .gitattributes to normalise end of line formatting
+* https://issues.couchbase.com/browse/NCBC-1762[NCBC-1762 -         Handle HTTP 429 for FTS
+* https://issues.couchbase.com/browse/NCBC-1766[NCBC-1766] -         Add support for FTS query retries when memory quota is reached
+* https://issues.couchbase.com/browse/NCBC-1768[NCBC-1768] -         Add automatic reties for FTS queries
```
Comment (ingenthr) [inline modules/ROOT/pages/relnotes-dotnet-sdk.adoc]: Typo.  Retries, not reties.

File: modules/ROOT/pages/relnotes-dotnet-sdk.adoc [hunk 4]
```diff
@@ -79,6 +79,45 @@ https://github.com/couchbase/couchbase-net-client/fork
...
+=== Fixed Issues
+
+* https://issues.couchbase.com/browse/NCBC-1661[NCBC-1661] -          SvcRestartAll-SUBDOC and  SvcRestart-SUBDOC fail on Ubuntu/Watson
+* https://issues.couchbase.com/browse/NCBC-1694[NCBC-1694] - List, map, etc operations do not create document if missing
+* https://issues.couchbase.com/browse/NCBC-1746[NCBC-1746] - QueryAsync times out after 75 seconds, even with extended timeout values
+* https://issues.couchbase.com/browse/NCBC-1763[NCBC-1763] - Closing cluster can cause exception in OrphanResponseLogger
+* https://issues.couchbase.com/browse/NCBC-1765[NCBC-1765] - Facet term values not being set
+* https://issues.couchbase.com/browse/NCBC-1770[NCBC-1770] - Config requests are using ChildOf relationships on finished spans
+* https://issues.couchbase.com/browse/NCBC-1772[NCBC-1772] -  Random is not thread-safe and should be wrapped in lock
+* https://issues.couchbase.com/browse/NCBC-1773[NCBC-1773] -         Default span tags are missing if using another Tracer (eg Jeager)
```
Comment (ingenthr) [inline modules/ROOT/pages/relnotes-dotnet-sdk.adoc]: should be e.g. I believe

File: modules/ROOT/pages/relnotes-dotnet-sdk.adoc [hunk 5]
```diff
@@ -79,6 +79,45 @@ https://github.com/couchbase/couchbase-net-client/fork
...
+* https://issues.couchbase.com/browse/NCBC-1762[NCBC-1762 -         Handle HTTP 429 for FTS
+* https://issues.couchbase.com/browse/NCBC-1766[NCBC-1766] -         Add support for FTS query retries when memory quota is reached
+* https://issues.couchbase.com/browse/NCBC-1768[NCBC-1768] -         Add automatic reties for FTS queries
+* https://issues.couchbase.com/browse/NCBC-1774[NCBC-1774] -         Add operation key as span tag
+* https://issues.couchbase.com/browse/NCBC-1779[NCBC-1779] -         Add bucket name and config to logging for DEBUG and TRACE
+</li>
+
+=== Fixed Issues
+
+* https://issues.couchbase.com/browse/NCBC-1661[NCBC-1661] -          SvcRestartAll-SUBDOC and  SvcRestart-SUBDOC fail on Ubuntu/Watson
```
Comment (ingenthr) [inline modules/ROOT/pages/relnotes-dotnet-sdk.adoc]: Does the spacing matter here?  Why do we have a lot of space on this line after the dash, and less afterwards.

File: modules/ROOT/pages/relnotes-dotnet-sdk.adoc [hunk 6]
```diff
@@ -79,6 +79,45 @@ https://github.com/couchbase/couchbase-net-client/fork
...
+http://packages.couchbase.com/clients/net/2.5/Couchbase-Net-Client-2.7.0.zip[Download] | http://docs.couchbase.com/sdk-api/couchbase-net-client-2.7.0[API Reference]
+
+=== Known Issues
+
+This release has the following known issues:
+
+* https://issues.couchbase.com/browse/NCBC-1296[NCBC-1296] -
+.NETCore SDK N1QL query throws System.PlatformNotSupportedException
+on some .NET Core platforms.  A workaround is available.
+* https://issues.couchbase.com/browse/NCBC-1502[NCBC-1502] -
```
Comment (ingenthr) [inline modules/ROOT/pages/relnotes-dotnet-sdk.adoc:93]: The description of this one leaves a bit to be desired.  Can we make it a bit more clear?

**Review by ingenthr — CHANGES_REQUESTED** [review]
A few improvements would make this much better

File: modules/ROOT/pages/relnotes-dotnet-sdk.adoc (hunk 2 above)
Comment (jeffrymorris) [reply modules/ROOT/pages/relnotes-dotnet-sdk.adoc:86]: I agree.

**Pull request closed without merging**